gridt/src/app/core/models/subscriptions.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Subscription } from 'rxjs';
+
+export class Subscriptions {
```

Comment:
I think the naming of this class could use some work. What do you think of SubscriptionHolder?

Reply:
I thought just subscriptions would be clear enough, since it is just the subscriptions. However i am also fine with the SubscriptionHolder name since that also acuratly describes what it does.